scripts/index-services.py

```diff
@@ -2,11 +2,11 @@
 """Read services from the API endpoint and write to search-api for indexing.
 
 Usage:
-    index-services.py <stage> --api-token=<api_access_token> --search-api-token=<search_api_access_token> [options]
+    index-services.py <stage> --frameworks=<frameworks> --api-token=<api_access_token> --search-api-token=<search_api_access_token> [options]
```

Comment:
hmm but then you get the newline appearing when you type the command, which is weird.

Looks like you can have line-continuation inside a comment, so let's see what that looks like (!!)